airflow/utils/helpers.py

```diff
@@ -21,7 +21,7 @@
 from datetime import datetime
 from functools import reduce
 from itertools import filterfalse, tee
-from typing import Any, Callable, Dict, Generator, Iterable, List, Optional, TypeVar
+from typing import TYPE_CHECKING, Any, Callable, Dict, Generator, Iterable, List, Optional, TypeVar
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Callable, Dict, Generator, Iterable, List, Optional, Tuple, TypeVar
```

This is needed to address the static check error caused by line 187

Reply:
Thanks.Added.